src/modules/commander/Commander.cpp

```diff
@@ -4293,7 +4293,6 @@ void Commander::manual_control_check()
 			// Abort autonomous mode and switch to position mode if sticks are moved significantly
 			// but only if actually in air.
 			if (manual_control_setpoint.sticks_moving
-			    && !_vehicle_control_mode.flag_control_manual_enabled
```

Comment:
Checking flag_control_manual_enabled is just for clarity/sanity, making it clear this entire block isn't even touched when in a manual mode.

Reply:
ok, I added it back. Doesn't hurt.